library/core/src/slice/mod.rs

```diff
@@ -2252,8 +2243,7 @@ impl<T> [T] {
     // in crate `alloc`, and as such doesn't exists yet when building `core`.
     // links to downstream crate: #74481. Since primitives are only documented in
     // libstd (#73423), this never leads to broken links in practice.
-    #[cfg_attr(not(bootstrap), allow(rustdoc::broken_intra_doc_links))]
-    #[cfg_attr(bootstrap, allow(broken_intra_doc_links))]
+    #[allow(broken_intra_doc_links)]
```

Comment:
This is wrong, it should have used the `not(bootstrap)` version. (It won't give a warning until 1.52 is the latest stable: https://github.com/rust-lang/rust/pull/83203).
```suggestion
    #[allow(rustdoc::broken_intra_doc_links)]
```